recipes-debian/watchdog/watchdog_debian.bb

```diff
@@ -0,0 +1,87 @@
+# base recipe: meta/recipes-extended/watchdog/watchdog_5.15.bb
+# base branch: warrior
+# base commit: 6d2e12e79211b31cdf5ea824fb9a8be54ba9a9eb
+
+SUMMARY = "Software watchdog"
+DESCRIPTION = "Watchdog is a daemon that checks if your system is still \
+working. If programs in user space are not longer executed \
+it will reboot the system."
+HOMEPAGE = "http://watchdog.sourceforge.net/"
+BUGTRACKER = "http://sourceforge.net/tracker/?group_id=172030&atid=860194"
+
+LICENSE = "GPL-2.0+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=ecc0551bf54ad97f6b541720f84d6569"
+
+require recipes-debian/sources/${BPN}.inc
+inherit debian-package
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/${BPN}-${PV}"
+DEBIAN_PATCH_TYPE = "nopatch"
+FILESPATH_append = ":${COREBASE}/meta/recipes-extended/watchdog/watchdog"
+SRC_URI += " \
+           file://0001-Include-linux-param.h-for-EXEC_PAGESIZE-definition.patch \
+           file://0001-watchdog-remove-interdependencies-of-watchdog-and-wd.patch \
+           file://watchdog.init \
+           file://wd_keepalive.init \
+"
+
+SRC_URI[md5sum] = "678c32f6f35a0492c9c1b76b4aa88828"
+SRC_URI[sha256sum] = "ffdc865137ad5d8e53664bd22bad4de6ca136d1b4636720320cb52af0c18947c"
+
+UPSTREAM_CHECK_URI = "http://sourceforge.net/projects/watchdog/files/watchdog/"
```

Comment:
meta-debian-extended uses debian source package. so it doesn't need to watch upstream source.

Reply:
I got it.